ts/src/wavesexchange.ts

```diff
@@ -2484,7 +2485,8 @@ export default class wavesexchange extends Exchange {
         let isErc20 = true;
         const noPrefix = this.remove0xPrefix (address);
         const lower = noPrefix.toLowerCase ();
-        for (let i = 0; i < lower.length; i++) {
+        const stringLength = lower.length * 1;
```

Comment:
because transpiler transpiled into `strlen` instead of `count` for php.

Reply:
@ttodua u sure?  this is the master version right now ⬇️ 
![image](https://github.com/ccxt/ccxt/assets/43336371/31df3076-6f1e-4907-8019-c1b8c2030eec)